config-linux.md

```diff
@@ -625,6 +625,22 @@ The following parameters can be specified to set up seccomp:
     * `SECCOMP_FILTER_FLAG_LOG`
     * `SECCOMP_FILTER_FLAG_SPEC_ALLOW`
 
+* **`listenerPath`** *(string, OPTIONAL)* - specifies the path of UNIX domain socket over which the runtime will send the [container process state](#containerprocessstate) data structure when the `SCMP_ACT_NOTIFY` action is used.
+    This socket MUST use `AF_UNIX` domain and `SOCK_STREAM` type.
+    The runtime MUST send exactly one [container process state](#containerprocessstate) per connection.
+    The connection MUST NOT be reused and it MUST be closed after sending a seccomp state.
+    If sending to this socket fails, the runtime MUST [generate an error](runtime.md#errors).
+    If the `SCMP_ACT_NOTIFY` action is not used this value is ignored.
```

Comment:
Shouldn't we send it no matter what? This can also be used as a "poor man's" mechanism of tracking container lifetime.

Reply:
I think it should not:
- so containers that don't use `SCMP_ACT_NOTIFY` policies would continue to work if the seccomp agent is not running or not installed.
- libseccomp automatically enables or disables `SECCOMP_FILTER_FLAG_NEW_LISTENER`  depending on whether we have at least one rule in the seccomp policy with `SCMP_ACT_NOTIFY`. So if container runtime implementations use libseccomp, they would not have a seccomp-fd to send without `SCMP_ACT_NOTIFY` rules. We make the implementation easier by following this (= no need to patch libseccomp).

Reply:
Yeah, I don't see any reason to allow it. Also, why not use the hooks for poor man's container lifecycle tracking?

Reply:
@sargun marking as solved as you recently reviwed with a LGTM. Please feel free to reopen if you think this isn't addressed :)